Classes/GlobalStateExplorers/DatabaseBrowser/FLEXSQLiteDatabaseManager.m

```diff
@@ -267,34 +272,34 @@ - (FLEXSQLResult *)errorResult:(NSString *)description {
     NSString *message = error ? @(error) : [NSString
         stringWithFormat:@"(%@: empty error)", description
     ];
-    
+
     return [FLEXSQLResult error:message];
 }
 
 - (id)objectForColumnIndex:(int)columnIdx stmt:(sqlite3_stmt*)stmt {
     int columnType = sqlite3_column_type(stmt, columnIdx);
-    
+
     switch (columnType) {
         case SQLITE_INTEGER:
-            return @(sqlite3_column_int64(stmt, columnIdx)).stringValue;
+            return [NSString stringWithFormat:@"%lld", sqlite3_column_int64(stmt, columnIdx)];
```

Comment:
Is this change necessary?

Reply:
in some cases, i got an exception for this. it's safe to change to stringWithFormat

Reply:
Do you have a crash log for that? I'm highly skeptical that could ever result in an exception/crash; `@()` should accept any return value and properly wrap it in `NSNumber`